custom/icds_reports/migrations/sql_templates/create_functions.sql

```diff
@@ -1186,6 +1186,10 @@ BEGIN
 		'usage_num_thr = ut.usage_num_thr, ' ||
 		'usage_num_hh_reg = ut.usage_num_hh_reg, ' ||
 		'is_launched = ut.is_launched, ' ||
+		'num_launched_states = ut.num_launched_awcs, ' ||
```

Comment:
@emord 
I had set num_launched_states / num_launched_districts / num_launched_blocks / num_launched_supervisors to null here since those values weren't applicable when viewing data per AWC.  Is it preferable to keep that, or was there a reason we want to set the value to 0?